packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -1030,6 +1038,35 @@ class RawScrollbar extends StatefulWidget {
   /// If null, defaults to Color(0x66BCBCBC).
   final Color? thumbColor;
 
+  /// The preferred smallest size the scrollbar can shrink to when the total
+  /// scrollable extent is large, the current visible viewport is small, and the
+  /// viewport is not overscrolled.
+  ///
+  /// The size of the scrollbar may shrink to a smaller size than [minThumbLength] to
+  /// fit in the available paint area. E.g., when [minThumbLength] is
+  /// `double.infinity`, it will not be respected if
```

Comment:
I am not sure I understand here. Should `double.infinity` be allowed as a `minThumbLength`? Also, the scrollbar currently only supports finite viewports. Infinitely long scrollables are not currently supported.